File: java/dry.java
public class dry {

	static void DoStuff() {
		System.out.println("Doing stuff!");
	}
	
	static int DoSum(int a, int b) {
		return a + b;
	}
	
    public static void main(String[] args) {
		DoStuff();
        System.out.println("2 + 1 = " + DoSum(2, 1));
    }

}

File: src/scala/dry.scala
object Dry extends App {
   
   def doStuff() {
       println("Doing stuff!")
   }

   def doSum(a:Int, b: Int) :Int = {
       return a + b
   }

   doStuff()
   println(s"2 + 1 = ${doSum(2, 1)}")
}